packages/cozy-harvest-lib/test/components/__snapshots__/AccountForm.spec.js.snap

```diff
@@ -158,6 +158,52 @@ exports[`AccountForm AccountFields should render encrypted fields with placehold
 </div>
 `;
 
+exports[`AccountForm should have disabled button if there is required field empty 1`] = `
+<div>
+  <AccountFields
+    fillEncrypted={false}
+    manifestFields={
+      Object {
+        "passphrase": Object {
+          "encrypted": true,
+          "required": false,
+          "type": "password",
+        },
+        "username": Object {
+          "default": "test",
+          "encrypted": false,
+          "required": true,
+          "role": "identifier",
+          "type": "text",
+        },
+      }
+    }
+    t={
+      [MockFunction] {
+        "calls": Array [
+          Array [
+            "accountForm.submit.label",
+          ],
+        ],
+        "results": Array [
+          Object {
+            "isThrow": false,
+            "value": "accountForm.submit.label",
+          },
+        ],
+      }
+    }
+  />
+  <DefaultButton
+    className="u-mt-2 u-mb-1-half"
+    disabled={false}
```

Comment:
_AccountForm should have disabled button if there is required field empty_

Shouldn't `disabled` be `true` ?